extensions/amp-embedly-card/validator-amp-embedly-card.protoascii

```diff
@@ -14,17 +14,33 @@
 # limitations under the license.
 #
 
-tags: {  # embedly-card
+tags: {  # embedly-card 1.0
   html_format: AMP
   tag_name: "SCRIPT"
+  satisfies: "amp-embedly-card 1.0"
+  excludes: "amp-embedly-card 0.1"
   extension_spec: {
     name: "amp-embedly-card"
+    version_name: "v1.0"
+    version: "1.0"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # embedly-card 0.1 and latest
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-embedly-card 0.1"
+  excludes: "amp-embedly-card 1.0"
+  extension_spec: {
+    name: "amp-embedly-card"
+    version_name: "v0.1"
     version: "0.1"
     version: "latest"
```

Comment:
We differentiate versions when adding validator rules with `satisifies`/`excludes` when the rules differ between versions. Since the two versions have `requires_usage` and `deprecated_allow_duplicates` in common (omitted values), their set of validation rules is identical. Therefore we can just do this (effectively the only modification in this file is to insert `version: "1.0"` between `"0.1"` and `"latest"`:
```suggestion
tags: {  # embedly-card
  html_format: AMP
  tag_name: "SCRIPT"
  extension_spec: {
    name: "amp-embedly-card"
    version: "0.1"
    version: "1.0"
    version: "latest"
```